Core/PSPLoaders.cpp

```diff
@@ -322,5 +323,41 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	pspFileSystem.Mount("umd0:", fs);
 
 	std::string finalName = ms_path + file + extension;
+
+	std::string homebrewName = PSP_CoreParameter().fileToStart;
+	std::size_t lslash = homebrewName.find_last_of("/");
+	homebrewName = homebrewName.substr(lslash +1);
+	std::string madeUpID = g_paramSFO.GenerateFakeID();
+
+	char title[1024];
+	sprintf(title, "%s : %s", madeUpID.c_str(), homebrewName.c_str());
```

Comment:
snprintf(title, sizeof(title), ....)

I know it's not gonna overflow but this one is also for consistency so you can search for sprintf to find potential overflow bugs...

Alternatively just go ahead and use StringFromFormat.